ruma-events/src/key/verification/start.rs

```diff
@@ -281,14 +317,18 @@ mod tests {
                 .unwrap()
                 .deserialize()
                 .unwrap(),
-            StartEventContent::MSasV1(MSasV1Content {
+            StartEventContent {
                 from_device,
                 transaction_id,
-                hashes,
-                key_agreement_protocols,
-                message_authentication_codes,
-                short_authentication_string,
-            }) if from_device == "123"
+                method: StartMethod::MSasV1(
+                    MSasV1Content {
```

Comment:
Please collapse these:

```suggestion
                method: StartMethod::MSasV1(MSasV1Content {
```